docker/tomcat_expedius/Dockerfile

```diff
@@ -1,23 +1,16 @@
 FROM tomcat:7
 
-# Expedius resource directory.
-RUN mkdir -p /var/lib/expedius
-RUN mkdir -p /var/lib/expedius/hl7
-RUN mkdir -p /var/lib/expedius/.appdata
-RUN mkdir -p /var/lib/expedius/.ssl
-RUN mkdir -p /var/lib/expedius/log
+# Fetch Expedius binary
+RUN curl https://bitbucket.org/colcamexdev/expediusbinary/downloads/Expedius-4.12.19.war --output ./Expedius.war
 
```

Comment:
Cannot get this to work even though it makes perfect sense.  The ADD resource doesnt seem to find the download and/or it doesnt wait for the curl process to complete.